### ISSUE TITLE
The Installer Will NOT install the app

### ISSUE THREAD

**Issue by derei** [Artemis-RGB/Artemis.Installer#2]
Windows 7 x64

Internet connection checked and working. VPN disabled. 

System.Exception: Failed to download binaries, please ensure you have a working internet connection.
SecureChannelFailure
   at async Task<string> Artemis.Installer.Services.InstallationService.DownloadBinaries(string version, IDownloadable downloadable, string branch)
   at async Task Artemis.Installer.Screens.Install.Steps.InstallationStepViewModel.Install()
   at async void Artemis.Installer.Screens.Install.Steps.InstallationStepViewModel.OnInitialActivate()+(?) => { } [1]
   at object System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, object args, int numArgs)
   at object System.Windows.Threading.ExceptionWrapper.TryCatchWhen(object source, Delegate callback, object args, int numArgs, Delegate catchHandler) ---> System.Net.Http.HttpRequestException: An error occurred while sending the request.
   at async Task Artemis.Installer.Extensions.HttpClientExtensions.DownloadAsync(HttpClient client, string requestUri, Stream destination, IDownloadable downloadable, CancellationToken cancellationToken)
   at async Task<string> Artemis.Installer.Services.InstallationService.DownloadBinaries(string version, IDownloadable downloadable, string branch) ---> System.Net.WebException: The request was aborted: Could not create SSL/TLS secure channel.
   at WebResponse System.Net.HttpWebRequest.EndGetResponse(IAsyncResult asyncResult)
   at void System.Net.Http.HttpClientHandler.GetResponseCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   --- End of inner exception stack trace ---

**Comment by RobertBeekman** [comment]
Hey,

Sorry to see you're having issues. On what version of Windows are you?
Also, can you confirm that you can access https://builds.artemis-rgb.com/binaries/master/20210724.1/ ?

**Comment by derei** [comment]
Windows 7 x64

Internet connection checked and working. VPN disabled.

Yes, I can access and download the binaries. The installer doesn't seem to have access.

**Comment by Zero3** [comment]
I have the same issue, and am also running Windows 7. Perhaps the installer is trying to download using a new and fancy HTTP/TLS standard that Windows 7 does not support?

**Comment by RobertBeekman** [comment]
This appears to be an issue with .NET's http client on Windows 7 and 8

I don't currently have a solution unfortunately. You can ofcourse manually download and install the binaries but I realise this is not much of a solution.

**Comment by Zero3** [comment]
@RobertBeekman I checked the server, and it only supports these 3 cipher suites over TLS 1.2:

````
TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256
````

These are too new for many devices. The first two are only supported from Windows 10 and later, and the third one is not supported by any version of Windows (yet). Here is the list of cipher suites supported by Windows 7: https://docs.microsoft.com/en-us/windows/win32/secauthn/tls-cipher-suites-in-windows-7

So long story short: If you want to support downloading using the .NET `HttpClient` on Windows 7 and 8, the download server needs to be reconfigured to enable at least one supported cipher suite.

**Comment by RobertBeekman** [comment]
Hi, I'm no expert but I tried expanding the supported cyphers. Could you test it out?

**Comment by Zero3** [comment]
@RobertBeekman The problem is still there, and the list of cipher suites supported by the server is still the same :(.

You can see for yourself here: https://www.ssllabs.com/ssltest/analyze.html?d=builds.artemis-rgb.com&hideResults=on. Scroll to Cipher Suites section and look under TLS 1.2.

It seems like the webserver is running nginx, so you are probably using a too restrictive `ssl_ciphers` setting in its config. You can see a working example from Mozille here: https://ssl-config.mozilla.org/.

**Comment by kadirhas** [comment]
Hello,

I have the same issue with windows 7 64 bit. Are there any solutions for this?

**Comment by diogotr7** [comment]
This should now be fixed, please test.

**Comment by Zero3** [comment]
Works for me now :)
